lib/ruby_lsp/requests/document_symbol.rb

```diff
@@ -108,6 +108,17 @@ def visit_class(node)
 
       sig { override.params(node: SyntaxTree::Command).void }
       def visit_command(node)
+        if node.message.value == "test"
+          argument = node.arguments.parts.first
+          test_name = argument.child_nodes.first.value
+          create_document_symbol(
+            name: test_name,
+            kind: :method,
+            range_node: argument,
+            selection_range_node: argument,
```

Comment:
The selection range should encapsulate the entire test, which matches the expected behaviour from the spec. For example, method definitions always include the body in the selection.